docs/podman-exec.1.md

```diff
@@ -38,6 +38,14 @@ Sets the username or UID used and optionally the groupname or GID for the specif
 The following examples are all valid:
 --user [user | user:group | uid | uid:gid | user:gid | uid:group ]
 
+**-w**, **--workdir**=""
+
+Working directory inside the container
+
+The default working directory for running binaries within a container is the root directory (/).
+The image developer can set a different default with the WORKDIR instruction. The operator
```

Comment:
I'd remove the second sentence, it's not really adding anything.  If you really want to keep it or massage it, please use the long form of the option `--workdir` rather than the short from.